src/main/java/com/pnudev/springonlinedelivery/dto/UserDto.java

```diff
@@ -0,0 +1,15 @@
+package com.pnudev.springonlinedelivery.dto;
+
+import com.pnudev.springonlinedelivery.models.Role;
+import lombok.Data;
+
+@Data
+public class UserDto {
+    private long id;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String mobile;
+    private String password;
```

Comment:
As this dto will be used in GET operations, you should not reveal the password here.
Think about a minimal meaningful amount of data that would be feasible to be presented on UI